### PR TITLE
Desktop Portfolio Website

### PR DESCRIPTION
**Pull request by MarkYegon7**
In this Project, I was able to:

- Follow the correct GitHub Flow principles.
- Use linters.
- Use Flexbox, CSS Grid, and media queries.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,34 +17,34 @@ <h2>Mark</h2>
...
-                <ul class="social-buttons">
-                    <li><a href="#"><img src="images/Vector.svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Group.svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (3).svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (4).svg" class="icons"></a></li>
-                    <li><a href="#"><img src="images/Vector (5).svg" class="icons"></a></li>
-                </ul>
+               <div class="ctn">
+                    <h1 class="title">I'm Mark <br>Glad to see you!</h1>
+                    <p class="intro">
```
Comment (Kossi-stack) [inline index.html:30]: - [ ] Kindly center well the header
- Your Design 
Screenshot 2023-05-12 at 11 10 27

-  Figma Design 
Screenshot 2023-05-12 at 11 15 22

File: styles.css [hunk 2]
```diff
@@ -9,721 +11,847 @@ body {
...
+    font-family: 'Poppins', sans-serif;
+    margin-top: 15px;
+    font-size: 1.563rem;
+    font-weight: 500;
+    height: 3.125rem;
+    margin-bottom: 0.938rem;
+    border-bottom: 0.063rem solid rgb(186, 183, 183);
+  }
+
+  .contact-form {
```
Comment (Kossi-stack) [inline styles.css:515]: - [ ] Kindly center the form well
- Your Design
Screenshot 2023-05-12 at 11 21 08

- Figma Design

Screenshot 2023-05-12 at 11 23 40

File: index.html [hunk 3]
```diff
@@ -17,34 +17,34 @@ <h2>Mark</h2>
...
     <section class="works-container" id="works-container">
         <div class="works">
-            <img src="images/Snapshoot Portfolio.svg" alt="" class="tonic">
-            <img src="images/Snapshoot Portfolio1.svg" alt="" class="tonic-1">
-            <div class="div1">
-                <h2 class="header-tonic">Tonic</h2>
+            <section class="illustration">
+            </section>
+            <div class="div1 w_ctn">
+                <h2 class="header">Tonic</h2>
```
Comment (Kossi-stack) [inline index.html:55]: - [ ] Kindly style the card well by adding some more padding to the `h2`
- Your Design
Screenshot 2023-05-12 at 11 25 58

- Figma Design
Screenshot 2023-05-12 at 11 29 24

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @MarkYegon7,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (MarkYegon7) [reply index.html:30]: Screenshot from 2023-05-13 00-03-05
This is how my project looks like.

The first reviewer marked another project. That is evident from line 21 which says I'm Mark compared to the screenshot provided by the reviewer.

File: styles.css (hunk 2 above)
Comment (MarkYegon7) [reply styles.css:515]: Screenshot from 2023-05-13 00-09-14
The contact form is centered.

File: index.html (hunk 3 above)
Comment (MarkYegon7) [reply index.html:55]: Screenshot from 2023-05-13 00-12-38

Padding is correctly placed

File: styles.css [hunk 4]
```diff
@@ -9,721 +11,847 @@ body {
...
+    background: url('images/contact\ form\ shapes-mobile.svg') no-repeat;
+    background-size: auto;
```
Comment (TadesseDev) [inline styles.css:513]: - [ ] dear @MarkYegon7, Just as mentioned by the previous reviewer, we need to center out form content appropriately, currently, your button and title are not aligned to the center as shown on the figma design. This is just to make sure we have a correct layout as given in the figma design. 

- [Optional] Please try to add the border-radius at the top of the from. As frontend engineers, we are expected to deliver something identical to the original design. Also, consider minimizing the width for your form elements as shown in the figma design. 
### Your design 
image
### Figma design 
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
x

File: index.html [hunk 5]
```diff
@@ -17,34 +17,34 @@ <h2>Mark</h2>
     </header>
```
Comment (divinecharlotte) [inline index.html:25]: - [ ] kindly as you fix the header as requested by the previous reviewer kindly consider sticking to the Figma design as much as possible by displaying all the nav items and giving the logo the same font size as given in the design to make your project an exact duplicate of the given design check the images below for more clarification 
- your design 👇 
Screen Shot 2023-05-12 at 23 34 29


- Figma design 👇 
Screen Shot 2023-05-12 at 23 36 23

File: index.html [hunk 6]
```diff
@@ -237,4 +238,4 @@ <h2 class="Contact">Contact me</h2>
     </section>
```
Comment (divinecharlotte) [inline index.html:246]: - [ ] Kindly as you center the form consider sticking to the Figma design by making sure that the `get in touch` button is also centered and also giving your paragraph a small width to make your project an exact duplicate of the provided design
- your design 👇 
Screen Shot 2023-05-12 at 23 42 34

- Figma design 👇 
Screen Shot 2023-05-12 at 23 42 01

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @MarkYegon7     , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- linters are passing 👍 
- Descriptive Pr 👍 




 

## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (MarkYegon7) [reply index.html:25]: Fixed the issue

File: index.html (hunk 6 above)
Comment (MarkYegon7) [reply index.html:246]: Made the required changes

File: index.html [hunk 7]
```diff
@@ -14,37 +14,45 @@ <h2>Mark</h2>
...
             <img src="images/Enabled.png" alt="" class="image-icon">
         </nav>
+
+        <nav>
+            <ul class="nav-content">
+                <li><a>Portfolio</a></li>
+                <li><a>About</a></li>
+                <li><a>Contact</a></li>
+              </ul>
+        </nav>
```
Comment (TEMESGEN-G25) [inline index.html:24]: - [ ] Kindly, as mentioned by the previous reviewer you should match the Figma design as much as possible. That includes margin, padding, color, images, & so on. Currently, your logo is missing proper margins/paddings.

## Your design
image

## Figma
image

File: index.html [hunk 8]
```diff
@@ -237,4 +246,4 @@ <h2 class="Contact">Contact me</h2>
     </section>
```
Comment (TEMESGEN-G25) [inline index.html:246]: - [ ] Great job :rocket: implementing the previous reviewer's suggestions, but, unfortunately, the Contact Me section isn't matching the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=MP8F21NcNgsWpIRH-0). The button hasn't been centered properly nor does the paragraph has the correct width.

**Review by TEMESGEN-G25 — CHANGES_REQUESTED** [review]
## :yellow_circle: CHANGES REQUIRED :yellow_circle: 

## Hi @MarkYegon7 :wave:, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
:heavy_check_mark: Implemented most of the previous reviewer suggestions.



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (MarkYegon7) [reply index.html:24]: Fixed the problem.

File: index.html (hunk 8 above)
Comment (MarkYegon7) [reply index.html:246]: Addressed the issue

File: index.html [hunk 9]
```diff
@@ -14,37 +14,45 @@ <h2>Mark</h2>
...
 
     <section class="works-container" id="works-container">
         <div class="works">
-            <img src="images/Snapshoot Portfolio.svg" alt="" class="tonic">
-            <img src="images/Snapshoot Portfolio1.svg" alt="" class="tonic-1">
-            <div class="div1">
-                <h2 class="header-tonic">Tonic</h2>
+            <section class="illustration">
+            </section>
+            <div class="div1 w_ctn">
```
Comment (Zuhaib042) [inline index.html:54]: - [OPTIONAL] There are some serious issues in your projects' cards. I will highly recommend that you fix them. these issues were missed by the previous reviewer.
   -  The heading in the card is left aligned in the figma design but in our design it is very poorly placed and dont give a good impression.
   - The  top and bottom margins of the elements in projects' description are not similar to figma design so consider adding more margin to make them identical to the figma design.

**Review by Zuhaib042 — APPROVED** [review]
Hi @MarkYegon7 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Linters passing
✅ Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**